t/Equaldex.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::Equaldex)],
+
+    'lgbt china' => test_spice(
+        '/js/spice/equaldex/china',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+
+    'lesbian rights cyprus' => test_spice(
+        '/js/spice/equaldex/cyprus',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'gay rights australia' => test_spice(
+        '/js/spice/equaldex/australia',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'transgender rights usa' => test_spice(
```

Comment:
*self-note* This test will fails return a result from the API as `usa` is passed it should be `United States`. Convert country codes to full country name.